pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -169,10 +169,34 @@ func (c *ClusterConfig) ValidateVPCConfig() error {
 		if *v != string(IPV4Family) && *v != string(IPV6Family) {
 			return fmt.Errorf("invalid value %s for ipFamily; allowed are %s and %s", *v, IPV4Family, IPV6Family)
 		}
+		// This is the new vpc check, I need this check when the user sets it.
+		if *v == string(IPV6Family) {
+			if missing := c.addonContainsManagedAddons([]string{"vpc-cni", "coredns", "kube-proxy"}); len(missing) != 0 {
```

Comment:
could we use consts for the addons? 😇

Reply:
I was so about to add it, but noticed that nothing else is doing that but call them by name and I was lazy. :DDDDDDD But for you, I'll add it :D